osmnx/save_load.py

```diff
@@ -169,11 +169,14 @@ def save_graph_osm(G, node_tags=settings.osm_xml_node_tags,
     if folder is None:
         folder = settings.data_folder
 
+    # get undirected graph so we don't generate duplicate nodes
+    H = get_undirected(G)
+
     # create a copy to convert all the node/edge attribute values to string
-    G_save = G.copy()
+    H_save = H.copy()
```

Comment:
Might be more efficient to say:
```suggestion
    H_save = get_undirected(G).copy()
```
instead of creating that `H` variable if it's unused elsewhere.

Reply:
done